errbot/bootstrap.py

```diff
@@ -63,6 +63,8 @@ def bot_config_defaults(config):
         config.TEXT_DEMO_MODE = True
     if not hasattr(config, 'BOT_ADMINS'):
         raise ValueError('BOT_ADMINS missing from config.py.')
+    if not hasattr(config, 'BOT_ADMINS_MUTE_NOTIFICATIONS'):
```

Comment:
I see your point on the positive flag.
It makes more sense to make it the list of admins to get notified. It simplifies all the logic too!
A side effect is that non-admins *could* potentially get notifications.